Classification/test/Classification/test_io.cpp

```diff
@@ -0,0 +1,100 @@
+#if defined (_MSC_VER) && !defined (_WIN64)
```

Comment:
I have an issue with this file:
```
-- Configuring test in test/Classification
-- Boost version: 1.66.0
-- Boost version: 1.66.0
-- Found the following Boost libraries:
--   serialization
--   iostreams
--   regex
-- Found Eigen3: /usr/include/eigen3 (found suitable version "3.3.5", minimum required is "3.1.0") 
-- Found Intel TBB
CMake Error at Installation/cmake/modules/CGAL_CreateSingleSourceCGALProgram.cmake:49 (add_executable):
  add_executable cannot create target "test_io" because another target with
  the same name already exists.  The existing target is an executable created
  in source directory
  "/home/lrineau/Git/other-cgal/Arrangement_on_surface_2/test/Arrangement_on_surface_2".
  See documentation for policy CMP0002 for more details.
Call Stack (most recent call first):
  Classification/test/Classification/CMakeLists.txt:108 (create_single_source_cgal_program)
```

Reply:
It should be renamed.

Reply:
@maxGimeno The `cmake` command in `.travis.yml` should use `-DWITH_examples=ON`, `-DWITH_demos=ON`, and `-DWITH_tests=ON`, to check that no two targets have the same names.

Reply:
Okay, I've used a more specific name.